cmd/ingress/ingress.go

```diff
@@ -71,19 +98,22 @@ func NewIngressCommand() *cobra.Command {
 			}()
 
 			router := pkg.Route()
-			err := http.ListenAndServe(":8080", router)
+			err = http.ListenAndServe(":8080", router)
```

Comment:
The `:8080` need to use `config.HTTPListen` instead.

Reply:
Sure, will be changed in next PR :).

This PR will not introduce any codes that use the new config.

Reply:
OK, understanding.